src/styles/index.scss

```diff
@@ -1,3 +1,64 @@
+@import './variables';
+
+// Загальні стилі для сторінки
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  box-sizing: border-box;
```

Comment:
It is a good practice to use 'border-box' for all elements in your project. This can be achieved by adding '*, *::before, *::after { box-sizing: border-box; }' to your CSS. This will help to avoid unexpected behavior in some cases.